src/pages/index.jsx

```diff
@@ -365,12 +365,12 @@ export async function getServerSideProps ({ query }) {
   // Filter out blacklisted entries
   // IF a valid config is provided
   if (
-    config.BLACKLISTED_CFTOOLS_IDS
-    && Array.isArray(config.BLACKLISTED_CFTOOLS_IDS)
-    && config.BLACKLISTED_CFTOOLS_IDS[0]
+    config.BLACKLISTED_STEAM64_IDS
+    && Array.isArray(config.BLACKLISTED_STEAM64_IDS)
+    && config.BLACKLISTED_STEAM64_IDS[0]
   ) {
     // Filter out blacklisted id's
-    const filteredRes = res.filter(({ id }) => !config.BLACKLISTED_CFTOOLS_IDS.includes(id));
+    const filteredRes = res.filter(({ id }) => !config.BLACKLISTED_STEAM64_IDS.includes(id));
```

Comment:
The `id` property is destructured from the CFTools API, the id represents a CFTools API id, **not** a steam64 - therefor, I have marked this pull request incorrect

Reply:
Here is proof the id property represents a CFTools id as this is clearly not a steam64 being all numbers.

![Code_jKLRnOhV3d](https://user-images.githubusercontent.com/57721238/214573007-ed554912-6350-49e9-911b-73ddb542e0da.png)